DetectorDescription/DDCMS/plugins/test/DDTestSpecParsFilter.cc

```diff
@@ -12,6 +12,12 @@ using namespace std;
 using namespace cms;
 using namespace edm;
 
+#ifdef HAVE_GEANT4_UNITS
+#define MM_2_CM 1.0
+#else
+#define MM_2_CM 0.1
+#endif
+
```

Comment:
@ianna @cvuosalo @civanch out of curiosity, I guess this is meant to be a temporary solution to help in the migration from one system of units to another, not as a long term approach to be used anywhere, am I correct?

Reply:
@fabiocos - yes, correct. It is also what DD4hep has implemented for the units migration.